zerver/tests/test_openapi.py

```diff
@@ -319,9 +319,7 @@ def get_type_by_priority(
         for t in types:
             if isinstance(t, tuple):
                 return t  # e.g. (list, dict) or (list, str)
-            v = priority.get(t, 6)
-            if v < val:
-                val = v
+            val = min(val, priority.get(t, 6))
```

Comment:
This is not a backport because it was deleted on main in f1d828cb7df7fe30ef6dbadb1b64b1f574ef08e5 before ruff could notice it.